core/templates/dev/head/forms/formBuilder.js

```diff
@@ -26,6 +26,20 @@ oppia.constant(
   'RTE_COMPONENT_SPECS',
   window.GLOBALS.RTE_COMPONENT_SPECS ? window.GLOBALS.RTE_COMPONENT_SPECS : {});
 
+// Service for retrieving width of the image
+oppia.factory('imageComponentWidthService', function() {
```

Comment:
This is a bit weird. Why is it a global service? That would imply all images have the same width...